bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJetsToLL_M-10to50_13TeV-madgraphMLM-pythia8/DYJetsToLL_M-10to50_13TeV-madgraphMLM-pythia8_run_card.dat

```diff
@@ -77,13 +77,13 @@ $DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf
 # Automatic ptj and mjj cuts if xqcut > 0
 # (turn off for VBF and single top processes)
 #**********************************************************
-   F  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
 #**********************************************************
 #                                                                    
 #**********************************
 # BW cutoff (M+/-bwcutoff*Gamma)
 #**********************************
-  15  = bwcutoff      ! (M+/-bwcutoff*Gamma)
```

Comment:
I don't remember exactly what is the story about the bw cutoff for this sample, is 15000 ok? @kdlong @qliphy